### PR TITLE
Add a 'projects' page

### PR DESCRIPTION
**Pull request by alexcn3**
Screenshot 2020-05-28 at 4 26 25 PM
Screenshot 2020-05-28 at 3 53 26 PM
Screenshot 2020-05-28 at 3 52 55 PM

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,31 +8,32 @@
   
   <div class="container-fluid text-center padded aboutme-para">
           
-      <div id="mySidenav">
-      <a href="#" id="projects"> Projects </a>
-      <a href="#" id="about"> About </a>
-      </div>
+    <div id="mySidenav">
```
Comment (eyh8) [inline portfolio/src/main/webapp/index.html]: Nit but we tend to use snake case for CSS selectors

Comment (alexcn3) [reply portfolio/src/main/webapp/index.html]: Just fixed it! Should I submit another pull request?

Comment (eyh8) [reply portfolio/src/main/webapp/index.html]: No, you can just push the changes to this branch. 😃

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,31 +8,32 @@
   
   <div class="container-fluid text-center padded aboutme-para">
           
-      <div id="mySidenav">
-      <a href="#" id="projects"> Projects </a>
-      <a href="#" id="about"> About </a>
-      </div>
+    <div id="my_sidenav">
```
Comment (eyh8) [inline portfolio/src/main/webapp/index.html]: `my-sidenav` instead?

**Review by eyh8 — APPROVED** [review]

**Pull request merged**